androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -71,6 +92,76 @@ fun EnhancedVideoPlayer(
         mutableStateOf(exoPlayer.currentMediaItem?.mediaMetadata?.displayTitle?.toString())
     }
 
+    val transition = rememberInfiniteTransition()
+
+    val scale = transition.animateFloat(
```

Comment:
Android Studio is complaining to me that this animation should have a label:

> The label parameter should be set so this infinite transition property can be better inspected in the Animation Preview.

![image](https://github.com/profusion/android-enhanced-video-player/assets/35314270/8c731a46-5d69-4e9b-8855-6d4b6385739a)

Reply:
in my version of Android studio this message doesn't appear, i'm not using the last version

Reply:
done